src/managers/PlayerManager.ts

```diff
@@ -15,6 +16,54 @@ class PlayerManager {
             this.onSync.emit();
         });
     }
+    static getPlayerId(): Nullable<string> {
+        let playerId: Nullable<string>;
+        playerId = localStorage.getItem("playerId");
```

Comment:
I dont see definition of localStorage or import anywhere, how does it work?
In my implementation, each Manager has its own "local storage":
SessionManager stores information from SessionManager in server (sessionID, playername, playerID, role, turnIndex)
PlayerManager stores list of playernames
(Just saw that controller in server looks completely different than when i wrote code 04.04 12:31)

Reply:
Outdated comment?

Reply:
yes, outdated comment